### PR TITLE
Fixed scrollbar bug in embedded form

### LINKED ISSUE #128: Can't scroll down google form on mobile
When I open the form to add a post on mobile, scrolling moves the close icon around a little but I can not scroll down the form.

Production:
screen shot 2016-08-08 at 4 03 35 pm

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #128

@acabunoc @cadecairos tiny PR for review!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -74,6 +70,10 @@
     </div>
   </footer>
 
+  <div id="add-project-form">
```
Comment (mmmavis) [inline index.html]: didn't change this part much except

1. moved this block down
2. changed it from `<section>` to `<div>`

File: index.html [hunk 2]
```diff
@@ -74,6 +70,10 @@
...
+    <iframe src="https://docs.google.com/forms/d/1CiNYA3gzHPd4HYrgQuQB2KM0TDpsbn6jbMcl-0_OjM4/viewform?embedded=true" width="100%" height="100%" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
+  </div>
```
Comment (mmmavis) [inline index.html]: didn't change this part much except

1. moved this block down
2. changed it from `<section>` to `<div>`

File: css/projects.css [hunk 3]
```diff
@@ -96,11 +96,12 @@ nav .container {
 
 #add-project-form {
   display:none;
-  position:fixed;
```
Comment (mmmavis) [inline css/projects.css]: on mobile, scrollbar doesn't show on fixed position element

**Comment by cadecairos** [comment]
Might be a Firefox Android specific bug, but if I open, close, and then re-open the submission form, it becomes unscrollable. Couldn't reproduce it in Chrome for Android.

**Comment by mmmavis** [comment]
Blah! I don't have an android device with me at the moment. Will check later today or tomorrow.

**Comment by mmmavis** [comment]
Okay, I was able to reproduce the FF on Android bug. I'm going to merge this PR and create a follow up ticket for that bug.

**Pull request merged**